templates/worker/00-worker/baremetal/files/dhcp-dhclient-conf.yaml

```diff
@@ -0,0 +1,7 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/dhcp/dhclient.conf"
+contents:
+  inline: |
+    supersede domain-search "{{ .EtcdDiscoveryDomain }}";
+    prepend domain-name-servers {{`{{ .NonVirtualIP }}`}};
```

Comment:
Who will render this? DId you run baremeta-runtimecfg over this file? IIRC the prepender script was in charge for that.

Reply:
> Who will render this? DId you run baremeta-runtimecfg over this file? IIRC the prepender script was in charge for that.

It's rendered directly. This is why I don't think we need the prepender anymore. It will be there from the first boot.

Reply:
resolved with non-virtual-ip-prepender